Create-NPM-Module.md

```diff
@@ -0,0 +1,51 @@
+##Create an NPM Module
+
+An NPM module is a set of Javascript functionality bundled into a distributable package.
+[NPM](http://www.npmjs.com) maintains the registry of all available packages, and is also the tool used to install
+packages from its registry. 
+
+The beauty of NPM is that you can assemble packages other people have created to create something new,
+and someone later on could use the package you create. If you have some code that you want to share with the world, 
+publishing a module to NPM is easy.
+
+####Step 1: Create your script
+
+I created this simple utility as `index.js`:
+
+```
+var time = new Date().toTimeString();
+console.log(time);
+```
+
+####Step 2: Create package information
```

Comment:
Same as [above](https://github.com/FreeCodeCamp/wiki/pull/884/files#r62623452). Add spacing between you're `##` and the actual title.